.cloudbuild/graal-build.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - name: oracle/graalvm-ce:latest
+    entrypoint: bash
+    args: ['./.cloudbuild/graal-build-script.sh']
+options:
+  machineType: 'N1_HIGHCPU_8'
```

Comment:
Specifying a machine type (at one point in the past) made the builds slower, because GCB doesn't have custom instances warmed up in their worker pool. I'm not sure if this is still the case, though if we need the extra RAM/CPU we may just need to bite the bullet.

Reply:
Ah yes, I changed the `machineType` here because I was getting OOM errors.

It does seem like the build runs decently fast though (faster than github actions) so maybe they did indeed fix the warm-up issue.